database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -20,7 +20,6 @@ public function up()
             $table->unsignedTinyInteger('role')->default(\App\User::USER); // default role user
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
-            $table->integer('role');
```

Comment:
- Where is the new migrate file?
- Where is using this column?

Reply:
yes, migrate create table user
columns role is already available, duplicates with develop branch so delete